x-pack/plugins/security_solution/scripts/endpoint/event_filters/index.ts

```diff
@@ -76,7 +81,30 @@ const createEventFilters: RunFn = async ({ flags, log }) => {
   await pMap(
     Array.from({ length: flags.count as unknown as number }),
     () => {
-      const body = eventGenerator.generate();
+      let options: Partial<ExceptionListItemSchema> = {};
+      if (flags.withWildcardPaths) {
+        const seed = parseInt((Math.random() * 3).toString(), 10);
+        const os = seed === 0 ? 'linux' : seed === 1 ? 'macos' : 'windows';
```

Comment:
Optional: You could also just expose the `.randomOSFamily()` method of the `eventGenerator` class and use it here